sale_start_end_dates/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
+Sale Start End Dates
```

Comment:
readme should be splitted according to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0